test/test_muscles.cpp

```diff
@@ -134,7 +136,7 @@ TEST(IdealizedActuator, copy)
 
         biorbd::muscles::Geometry geometry(idealizedActuator.position());
         geometry.setInsertionInLocal(biorbd::utils::Vector3d(0.5, 0.6, 0.7));
-        
+
         //Can't freakin update the orientations!!! TODO
         //idealizedActuator.updateOrientations(model, Q, QDot, 2);
```

Comment:
What do you mean?
Please don't comment non working code since the goal is that I can see the error!

Reply:
I change the geometry of the idealizedActuator to see if it also changes the geometry of its shallow copy. But everytime I try to use updateOrientations, it gives me an error and doesn't go through. I wrote TODO for myself, because I wanted to advance other parts of the tests.

Reply:
As I said, this is a bug, that myself have to solve ;)
Copying the muscle somehow deletes something which in turn create a SegFault, this is kind of a major bug that you found :P

Always make sure that if something fails that the tests don't pass! 

I'll solve that and push the updated version!

Reply:
Ok, no I found the problem, you can't send a Vector3d to setInsertionInLocal, it has to be either a viapoint or a node3d, I added the checker to have a proper error message (and prevent the segfaulting)
thanks for finding this!

Reply:
ok ok.. finally everything was okay in the code... haha
Using Vector3d, you were actually actively changing the name of the parent of the insertion point. Since there were no parent left, it failed when it came to find its position relative to the parent

Reply:
The next push solves the problem from the test point of view. Unfortunately checking this in the code would slow it down too much... So no checks were added

Reply:
Last thing : make sure you use camelCase for the variable names 
You can resolve :)